csharp/test/Ice/operations/MyDerivedClassAMDI.cs

```diff
@@ -1,829 +1,825 @@
 // Copyright (c) ZeroC, Inc.
 
-namespace Ice
+namespace Ice.operations.AMD;
```

Comment:
Is there any fixes in this file other than switching to a file scoped namespace?

Reply:
Yes, removed async and `await Task.Delay(0)`.